lib/imdb/search.rb

```diff
@@ -23,11 +32,13 @@ def movies
     private
 
     def document
-      @document ||= Nokogiri::HTML(Imdb::Search.query(@query))
+      @document ||= Nokogiri::HTML(submit_query)
     end
 
-    def self.query(query)
-      open("http://akas.imdb.com/find?q=#{CGI.escape(query)};s=tt")
+    def submit_query
+      url = "http://akas.imdb.com/find?q=#{CGI.escape(query)}&s=tt"
```

Comment:
I changed the `;` separator to `&` because that's what happens when you use the IMDB website now.